### DISCUSSION TITLE
List products with redirections in the Monitoring page

### DISCUSSION THREAD

**Discussion by ghost** [PrestaShop/PrestaShop#40197] [Ideas]
**Is your feature request related to a problem? Please describe.**
Today you can redirect a product URL to another URL (product or category). 
To do so you have to do it in the tab "SEO" in the product page.

The problem is that there is no list of all the products that have a redirection, which means that to know which product is still redirected you have to go into each product individually **-> This takes a tremendous amount of time**

![Capture d’écran 2019-10-30 à 11 45 26](https://user-images.githubusercontent.com/47456895/67851453-c97a7000-fb0a-11e9-8e6e-e6fbcf83920e.png)
_in Catalog > Products > Add New/Edit > SEO tab_

**Describe the solution you'd like**
Add a table in the Monitoring page to list all the product being redirected at the present time and being able to disable the redirection from there, or to access the product page from there)

![Capture d’écran 2019-10-30 à 11 49 03](https://user-images.githubusercontent.com/47456895/67851705-4ad20280-fb0b-11e9-9d2d-510fcc37b305.png)
_Monitoring page_

**Describe alternatives you've considered**
None

**Comment by hibatallahAouadni** [comment]
Ping @PrestaShop/product-team shall I change the issue status to **To Do**, does this issue still need some specifications?
